### PR TITLE
fix: update releasing.md

### PR DESCRIPTION
**Pull request by mdonnalley**
@W-10415754@

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -45,17 +47,17 @@ In the Heroku CLI, we have it automatically build and release the beta channel o
 
 ## Windows installer
 
-Build a windows installer with `oclif pack:win`. It will build into `./dist/win`. This can be published to S3 with `oclif promote --win`.
+Build a windows installer with `oclif pack:win`. It will build into `./dist/win`. This can be uploaded to S3 with `oclif upload:win` and promoted within S3 with `oclif promote --win`.
```
Comment (peternhale) [inline docs/releasing.md:50]: replace `oclif pack:win` with `oclif pack win` `oclif upload:win` with `oclif upload win` I know ':' still works but use space for consistency.

File: docs/releasing.md [hunk 2]
```diff
@@ -45,17 +47,17 @@ In the Heroku CLI, we have it automatically build and release the beta channel o
...
 
 ## macOS installer
 
-Build a macOS .pkg installer with `oclif pack:macos`. It will build into `./dist/macos`. This can be published to S3 with `oclif promote --macos`. You need to set the macOS identifier at `oclif.macos.identifier` in `package.json` (we use "com.heroku.cli" and "com.salesforce.cli" as the identifiers for the Heroku CLI and the Salesforce CLI, respectively).
+Build a macOS .pkg installer with `oclif pack:macos`. It will build into `./dist/macos`. This can be uploaded to S3 with `oclif upload:macos` and promoted within S3 with `oclif promote --macos`. You need to set the macOS identifier at `oclif.macos.identifier` in `package.json` (we use "com.heroku.cli" and "com.salesforce.cli" as the identifiers for the Heroku CLI and the Salesforce CLI, respectively).
```
Comment (peternhale) [inline docs/releasing.md:54]: Ditto `s/:/ /g`

File: docs/releasing.md [hunk 3]
```diff
@@ -45,17 +47,17 @@ In the Heroku CLI, we have it automatically build and release the beta channel o
...
 
 To [sign the installer](https://developer.apple.com/developer-id/), set `oclif.macos.sign` in `package.json` to a certificate (For the Heroku CLI this is "Developer ID Installer: Heroku INC"). And optionally set the keychain with `OSX_KEYCHAIN`.
 
 ## Ubuntu/Debian packages
 
-Build a deb package with `oclif pack:deb`. Set the `MYCLI_DEB_KEY` to a gpg key id to create the gpg files. This will include all the files needed for an apt repository in `./dist/deb`. They can be published to S3 with `oclif promote --deb`.
+Build a deb package with `oclif pack:deb`. Set the `MYCLI_DEB_KEY` to a gpg key id to create the gpg files. This will include all the files needed for an apt repository in `./dist/deb`. They can be uploaded to S3 with `oclif upload:deb` and promoted within S3 using `oclif promote --deb`.
```
Comment (peternhale) [inline docs/releasing.md:60]: Ditto

**Review by peternhale — CHANGES_REQUESTED** [review]

**Review by peternhale — APPROVED** [review]

File: docs/releasing.md [hunk 4]
```diff
@@ -19,13 +19,15 @@ $ npx mynewcli
...
 
-To publish, you can copy the files from `./dist` or use `oclif promote` to copy the files to S3. You'll need to set `oclif.update.s3.bucket` in `package.json` to a valid S3 bucket and have credentials set in `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment vars.
+To publish, you can copy the files from `./dist` or use `oclif upload tarballs` to copy the files to S3. You'll need to set `oclif.update.s3.bucket` in `package.json` to a valid S3 bucket and have credentials set in `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment vars.
+
+After you've uploaded the tarballs to S3, you can promote the tarballs to a different release channel within S3 using `oclif promote`. This allows you to quickly promote and demote a specific version between your various release channels. For example, the Salesforce CLI has separate `stable` and `stable-rc` channels that are updated weekly.
```
Comment (RodEsp) [inline docs/releasing.md:30]: `oclif upload tarballs` doesn't create a release channel. It just uploads the tarballs to a directory derived from its intended target.
```suggestion
After you've uploaded the tarballs to S3, you can promote the tarballs to a release channel within S3 using `oclif promote`. This allows you to quickly promote and demote a specific version between release channels. For example, the Salesforce CLI has separate `stable` and `stable-rc` channels that are updated weekly.
```

Comment (peternhale) [reply docs/releasing.md:30]: Should these docs have a reference to Salesforce CLI as an example?

**Review by peternhale — CHANGES_REQUESTED** [review]

File: docs/releasing.md (hunk 4 above)
Comment (RodEsp) [reply docs/releasing.md:30]: @peternhale I don't see why not. It's just an example. We could reference other oclif based CLIs if you have any suggestions.

**Review by peternhale — APPROVED** [review]

**Pull request merged**